cmd/neofs-node/control.go

```diff
@@ -18,24 +19,24 @@ import (
 
 func initControlService(c *cfg) {
 	strKeys := controlconfig.AuthorizedKeysString(c.appCfg)
-	keys := make([][]byte, 0, len(strKeys)+1) // +1 for node key
+	pubs := make([][]byte, 0, len(strKeys)+1) // +1 for node key
 
-	keys = append(keys, crypto.MarshalPublicKey(&c.key.PublicKey))
+	pubs = append(pubs, c.key.PublicKey().Bytes())
 
 	for i := range strKeys {
 		key, err := hex.DecodeString(strKeys[i])
 		fatalOnErr(err)
 
-		if crypto.UnmarshalPublicKey(key) == nil {
-			fatalOnErr(fmt.Errorf("invalid permitted key for Control service %s", strKeys[i]))
+		if _, err := keys.NewPublicKeyFromBytes(key, elliptic.P256()); err != nil {
```

Comment:
I think the better way is to parse these keys in `controlconfig.AuthorizedKeys`. For the same reason we parse private key in config as well.